boards/seeedstudio-xiao-nrf52840/mtd.c

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2024 TU Dresden
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_seeedstudio-xiao-nrf52840
+ * @{
+ *
+ * @file
+ * @brief       MTD configuration for the XIAO nRF52840
+ *
+ * @author      Mikolai Gütschow <mikolai.guetschow@tu-dresden.de>
+ *
+ * @}
+ */
+
+#ifdef MODULE_MTD
+
+#include "board.h"
+#include "mtd.h"
+#include "mtd_spi_nor.h"
+#include "periph_conf.h"
+#include "timex.h"
+
+static const mtd_spi_nor_params_t _xiao_nrf52840_nor_params = {
+    .opcode = &mtd_spi_nor_opcode_default,
+    /* datasheet reports 20ms, but experiments showed ~135ms*/
+    .wait_chip_erase = 200LU * US_PER_MSEC,
```

Comment:
```suggestion
    .wait_chip_erase = 200LU * US_PER_MS,
```

The macro `US_PER_MSEC` is undefined and leads to a compile error. Small typo :)